index.js

```diff
@@ -1,102 +1,28 @@
-/* eslint max-classes-per-file: ["error", 2] */
-const booksList = document.getElementById('books-list');
+import { Book, Books } from './modules/books.js';
+// eslint-disable-next-line import/extensions
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+import {
+  showForm, showContact, showList, listLink, formLink, contactLink, formSection, contact,
+} from './modules/navigation.js';
+
 const form = document.getElementById('form');
 const title = document.getElementById('title');
 const author = document.getElementById('author');
 const time = document.getElementById('time');
-const list = document.getElementById('list');
-const formSection = document.getElementById('form-section');
-const contact = document.getElementById('contact');
-const listLink = document.getElementById('list-link');
-const formLink = document.getElementById('form-link');
-const contactLink = document.getElementById('contact-link');
-
-formSection.style.display = 'none';
-contact.style.display = 'none';
-
-function showForm() {
-  formSection.style.display = 'block';
-  contact.style.display = 'none';
-  list.style.display = 'none';
-}
-
-function showContact() {
-  contact.style.display = 'block';
-  formSection.style.display = 'none';
-  list.style.display = 'none';
-}
-
-function showList() {
-  list.style.display = 'block';
-  formSection.style.display = 'none';
-  contact.style.display = 'none';
-}
-
-formLink.addEventListener('click', showForm);
-contactLink.addEventListener('click', showContact);
-listLink.addEventListener('click', showList);
 
 setInterval(() => {
-  const date = new Date();
-  time.innerHTML = date.toLocaleTimeString();
+  time.innerHTML = DateTime.now().toLocaleString(DateTime.DATETIME_FULL);
 }, 1000);
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=the%20correct%20way.-,Display%20the%20current%20date%20below%20the%20navigation%20bar,-.), Kindly display the current date below the navbar.

![image](https://user-images.githubusercontent.com/68796232/228022317-79243e7b-ab57-4074-be19-6f8112db05ef.png)

Reply:
The date will not display unless you have luxon in the node modules.